README.md

```diff
@@ -107,11 +107,20 @@ You probably need to set some environment variables for the client to be useful.
   - Disabled by default.
   - Set to `true` enable.
 
+## Troubleshooting
+
+### The UI shows more sensors than I have
+
+The firmware reads all 12 ADC pins on the Teensyduino. You may get fake readings from pins that have no sensors plugged in and are not grounded. If pressing a sensor causes more than one value to go up:
+  - Determine which pins are unused. You can use this [pinout](https://www.pjrc.com/teensy/card2a.pdf)
```

Comment:
Perhaps it would be worth mentioning the that you should follow the ADC channels in incremental order (ADC0 = Sensor 1, but don't mind the missing ADC2 and 3 and just keep going).

Although it would probably not be a bad thing to modify the firmware to use the Arduino order because it perhaps makes more sense for user (this one https://www.pjrc.com/teensy/card2b.pdf).

Reply:
Added a note about incremental order.